### ISSUE TITLE
todo: add moguri videos and moguri music checkboxes in launcher

### ISSUE THREAD

**Issue by snouz** [Albeoris/Memoria#307]

**Comment by Tirlititi** [comment]
They can rather be added in the Mod Manager, either as separate mods or in a better way as Moguri options (just like Mog Add-ons's options). This can be achieved with [a couple of lines](https://github.com/Albeoris/Memoria/wiki/Mod-Manager) in Moguri's ``ModDescription.xml``.

**Comment by snouz** [comment]
The goal was to make sure these options are always visible, as they are in Moguri without an updated memoria.
I want to make sure people understand that these are things that stray away from the original experience.

In my commit, I made sure that closing the mod manager window will update the main launcher options.

**Comment by faospark** [comment]
oh my i can no longer detect following mods s the current released version of memoria 
![Screenshot_689](https://github.com/Albeoris/Memoria/assets/11664809/207224e5-8fd3-4209-9b2e-b81ff674ea2b)
[https://github.com/Albeoris/Memoria/releases/tag/v2023.06.11](https://github.com/Albeoris/Memoria/releases/tag/v2023.06.11)

MoguriSoundtrack
MoguriVideo

I verified the files and installed the current release of memoris. 

this is just based on my recollection. For many years. If memoria patches the game with a Moguri version installed 
3 folders would appear to the mod manger 
This is main moguri mod Which is in the folder Moguri Files 
Moguri Soundtrack and Moguri Video appearing separate mods in the mod manager.  That has been the case since Memoria became a mod manager for a couple of years now.   
if i would guess just based on the downloads of nexus most people are used for a long while now and if they open their game they might be surprised . 

I agree with @Tirlititi 's approach. its better to treat them as submods rather than putting the options on top of mod manager 
as this bring another issue of people not differentiating the memoria the mod manager/ engine vs Moguri.  i think strategic and wise to separate the mods from the mod manager . 

To summarize right now moguri sound track and video are not detected in the latest release of the mod manager and cant be changed not unless one changes it on the ini file. 

Just in case someone encounters the current predicament here is my not so elegant fix 
the fix and work around i did is to copy both MoguriSoundtrack and MoguriVideo folders to MoguriFiles folder and removed thier corresponding ModDescription.xml. Then configured them as a submod on MoguriFiles' ModDescription.xml.
here is a sample below  
[ModDescription.zip](https://github.com/Albeoris/Memoria/files/14342209/ModDescription.zip)

**Comment by snouz** [comment]
> oh my i can no longer detect following mods s the current released version of memoria
> Screenshot_689
> https://github.com/Albeoris/Memoria/releases/tag/v2023.06.11
> 
> MoguriSoundtrack
> MoguriVideo
> 
> I verified the files and installed the current release of memoris.
> 
> this is just based on my recollection. For many years. If memoria patches the game with a Moguri version installed
> 3 folders would appear to the mod manger
> This is main moguri mod Which is in the folder Moguri Files
> Moguri Soundtrack and Moguri Video appearing separate mods in the mod manager. That has been the case since Memoria became a mod manager for a couple of years now.
> if i would guess just based on the downloads of nexus most people are used for a long while now and if they open their game they might be surprised .

Could you test again with my latest commit, and remove moddescription.xml in MoguriVideo and MoguriSoundtrack? You might have downloaded the version where I replaced the content of these files with a new content, but it was missing a closing tag.

The intended (and current) behavior is that: the launcher has 2 tickboxes only if moguri is there, and they behave/are synchronized with the list in the mod manager for these 2 submods. You can still activate and deactivate from mod manager, it's just that it becomes more visible for users using moguri. This reflects the fact that the current launcher that moguri is using is custom, with an option for these. The end goal is to stop using a custom launcher for moguri, and just use memoria, that you could update as you wish. These options need to be visible for new moguri users, because they are installed by default, and if not visible, they'll become standard, and people need to see that it's not necessary standard (I personally consider changing the music a mistake, but to each their own). People who newly install the mod (and the current advice I see around is to install moguri, then patch with the latest memoria version), if they have to go to a separate window and manually disable this, this means new players will think the orchestral music is the original music, and the 30fps videos are original too.

To make sure people understand that this is a feature of moguri, I've added the mention "(moguri)".
If you have a better suggestion or disagree, I'm of course 100% open to it, I hope the reason I wanted to add this is clear. But I think a part of what you disagree with is due to the bug above
